Resources/Prototypes/Entities/Clothing/OuterClothing/armor.yml

```
@@ -21,6 +21,9 @@
         Heat: 0.80
   - type: ExplosionResistance
     damageCoefficient: 0.90
+  - type: TemperatureProtection # Mono
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)